cypress/e2e/register/register.spec.ts

```diff
@@ -85,4 +85,48 @@ describe("Registration", () => {
         cy.get(".mx_DevicesPanel_myDevice .mx_DevicesPanel_deviceTrust .mx_E2EIcon")
             .should("have.class", "mx_E2EIcon_verified");
     });
+
+    it("should require username to fulfil requirements and be available", () => {
```

Comment:
TIL "fulfil" is the UK spelling and "fulfill" is the US spelling. Another word to remember is different 😅